src/display/network.js

```diff
@@ -273,16 +273,17 @@ class PDFNetworkStreamFullRequestReader {
     const fullRequestXhrId = this._fullRequestId;
     const fullRequestXhr = this._manager.getRequestXhr(fullRequestXhrId);
 
-    const responseHeaders = new Headers(
-      fullRequestXhr
-        .getAllResponseHeaders()
-        .trim()
-        .split(/[\r\n]+/)
-        .map(x => {
-          const [key, ...val] = x.split(": ");
-          return [key, val.join(": ")];
-        })
-    );
+    const rawResponseHeaders = fullRequestXhr.getAllResponseHeaders();
+    const responseHeadersMap = rawResponseHeaders
+      ? rawResponseHeaders
+          .trim()
+          .split(/[\r\n]+/)
+          .map(x => {
+            const [key, ...val] = x.split(": ");
+            return [key, val.join(": ")];
+          })
+      : [];
+    const responseHeaders = new Headers(responseHeadersMap);
```

Comment:
The name `responseHeadersMap` feels a little strange here, since it's not a `Map` but actually an `Array`.
Anyway, that temporary variable shouldn't be needed if this is just re-written as follows.
```suggestion
    const responseHeaders = new Headers(
      rawResponseHeaders
        ? rawResponseHeaders
            .trim()
            .split(/[\r\n]+/)
            .map(x => {
              const [key, ...val] = x.split(": ");
              return [key, val.join(": ")];
            })
        : []
    );
 ```